templates/base.html

```diff
@@ -137,7 +137,37 @@
         <script src="//cdn.ravenjs.com/3.0.4/raven.min.js"></script>
         <script>Raven.config('{% sentry_public_dsn %}').install()</script>
         {% block script %}{% endblock %}
-
+        <!-- Piwik -->
+<script type="text/javascript">
+  var _paq = _paq || [];
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u="//organize-analytics.herokuapp.com/";
+    _paq.push(['setTrackerUrl', u+'piwik.php']);
+    _paq.push(['setSiteId', '2']);
+      var upenn_registration = '{{ request.session.is_upenn_registration }}';
```

Comment:
@nasief This line indentation doesn't match other lines in the same scope. It will not cause issues, just for code style purposes.